pages/projects/dashboard/index.js

```diff
@@ -364,7 +364,7 @@ export default function MscaDashboard(props) {
                   : pageData.scFragments[3].scContentFr.json[0].content[0]
                       .value}
               </h2>
-              <p className="col-span-12 xl:col-span-8">
```

Comment:
So from what I understand, we want to try and keep our line length around 65~ characters and no more than 80~. This is why on desktop generally we have our text span 8 out of the 12 columns.

Here's the research from the DS team:

https://www.figma.com/file/XkfHmBPKDcXKkrDVIZFlyF/DS-research-(Copy)?type=whiteboard&node-id=134-10&t=XB93hQUBy11s12gu-4